hooks/pre_commit/20-certs_update.rb

```diff
@@ -0,0 +1,18 @@
+if module_enabled?('certs')
+  if app_value('certs_update_server_ca') && !module_enabled?('katello')
+    fail_and_exit("--certs-update-server-ca needs to be used with katello", 101)
+  end
+
+  ca_file   = param('certs', 'server_ca_cert').value
+  cert_file = param('certs', 'server_cert').value
+  key_file  = param('certs', 'server_key').value
+
+  if !app_value('certs_skip_check') && cert_file.to_s != "" &&
+     (app_value('certs_update_server_ca') || app_value('certs_update_server'))
+    check_cmd = %(katello-certs-check -c "#{cert_file}" -k "#{key_file}" -b "#{ca_file}")
+    output = `#{check_cmd} 2>&1`
+    unless $CHILD_STATUS.success?
```

Comment:
This line is why there was a `require 'English'` in the other file. However, I'd suggest using `execute` from our hooks which has a lot more functionality built into it for running commands and handling failures.